### PR TITLE
Make it possible to run tests only

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

File: ansible_role/defaults/main.yml [hunk 1]
```diff
@@ -164,3 +164,5 @@ quay_registry:
 
 # conditionally set later based on Cluster configuration
 default_registries: []
+
+tests_deploy_broker: true
```
Comment (djzager) [inline ansible_role/defaults/main.yml]: Looking at the failure from travis. I think it's more correct to take this out of this file and add it to the vars of the ansible_role/playbooks/test.yml

Comment (jmontleon) [reply ansible_role/defaults/main.yml]: Understood. I'll do so in a few and see if that gets it cleaned up.

**Pull request merged**